### PR TITLE
Update build script and README to Thrift version 0.11.0

### PR DESCRIPTION
**Pull request by jafingerhut**
This version has been officially supported for a while now, and will
likely help in moving from Python2 to Python3.

### CODE REVIEW

File: install_deps.sh [hunk 1]
```diff
@@ -6,6 +6,8 @@ then
     # This older package libssl1.0-dev enables compiling Thrift 0.9.2
     # on Ubuntu 18.04.  Package libssl-dev exists, but Thrift 0.9.2
     # fails to compile when it is installed.
+    # TBD whether using this package makes a difference for Ubuntu
+    # 18.04 and Thrift 0.11.0.
```
Comment (jafingerhut) [inline install_deps.sh:10]: Let me know if anyone considers it important to determine whether we can use same libssl-dev package across Ubuntu 16.04 and 18.04 here.  The script as written works, but might also work with a common package for all supported Ubuntu versions.

**Review by antoninbas — APPROVED** [review]
LGTM

File: install_deps.sh (hunk 1 above)
Comment (antoninbas) [reply install_deps.sh:10]: as long as it works :)

**Comment by adhicode** [comment]
Is Thrift 0.13.0 supported? When i built BMv2 with thrift-0.13.0, i could run successfully "make check". Pls specify if we have any impact. Thanks

**Pull request merged**